lib/sqlalchemy/ext/mypy/plugin.py

```diff
@@ -89,10 +74,29 @@ def get_class_decorator_hook(
 
         return None
 
-    def get_customize_class_mro_hook(
+    def get_metaclass_hook(
```

Comment:
**mike bayer** (zzzeek) wrote:

ah, tests added for this?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2962